src/features/CourtCaseDetails/Tabs/Panels/Offences/Offence/HearingResult.tsx

```diff
@@ -18,6 +17,8 @@ import {
 } from "../../../../../../types/Amendments"
 import { Exception } from "../../../../../../types/exceptions"
 import { TableRow } from "../../TableRow"
+import EditableFieldTableRow from "../../../../../../components/EditableFieldTableRow"
```

Comment:
This file `src/features/CourtCaseDetails/Tabs/Panels/Offences/Offence/HearingResult.tsx` is starting to get unruly.

Reply:
Yeah, we need to do a refactor in the component. Do you have any suggestions?

Reply:
Create a ticket to refactor this.